include/smtlib_reader.h

```diff
@@ -150,6 +150,17 @@ class SmtLibReader
    */
   virtual void pop(uint64_t num = 1);
 
+  /** Mark a term with an attribute
+   *  This currently is unimplemented and just prints a warning.
+   *  It can be overridden in a derived class to do something
+   *  useful with the attribute
+   *  @param sym the symbol
+   *  @param term the term
```

Comment:
Can you update these?

Reply:
Oops, thanks for catching that! Fixed in 793be8ca80bf09cf834920d55f3564cdc93cdb64.